### PR TITLE
[BPK-862] Android platform targeting for tokens

### PR DESCRIPTION
**Pull request by shaundon**
Added minimal tokens for text component, some custom logic for
displaying colours and font sizes in the correct format for Android.
Also added a test to verify that Android and iOS platforms are
implementing the correct tokens, taking platform specificity into
account.

+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [x] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ _Nothing yet!_
 **Fixed:**
 - bpk-tokens:
   - Remove unused output formats for iOS and Android which aren't relevant when the platform is web.
-  - Added new outputs for the iOS platform in `bpk-token/tokens/ios/base.ios.json` and `bpk-tokens/tokens/react.native.es6.js`. These interface are not to be considered stable yet however and they may still change.
+  - Added new outputs for the iOS and Android platforms in `bpk-tokens/tokens/ios/base.ios.json`, `bpk-tokens/tokens/react.native.es6.js`, `bpk-tokens/tokens/android/base.android.xml` and
```
Comment (k0nserv) [inline changelog.md]: Two of these paths aren't corret

Comment (shaundon) [inline changelog.md]: Whoops, fixed.

File: changelog.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ _Nothing yet!_
 **Fixed:**
 - bpk-tokens:
   - Remove unused output formats for iOS and Android which aren't relevant when the platform is web.
-  - Added new outputs for the iOS platform in `bpk-token/tokens/ios/base.ios.json` and `bpk-tokens/tokens/react.native.es6.js`. These interface are not to be considered stable yet however and they may still change.
+  - Added new outputs for the iOS and Android platforms in `bpk-tokens/tokens/ios/base.ios.json`, `bpk-tokens/tokens/ios/react.native.es6.js`, `bpk-tokens/tokens/android/base.android.xml` and
```
Comment (k0nserv) [inline changelog.md]: The `react.native.es6.sj` names should be `base.react.native.es6.js`

**Comment by tvooo** [comment]
@k0nserv can you approve? BTW sorry @shaundon, I created a new commit instead of squashing them, removing you from the history. Stealing your credits. Won't happen again :)

**Pull request merged**